modules/images-pulling-from-private-registries.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+// * openshift_images/using-image-pull-secrets
+
+[id="images-pulling-from-private-registries_{context}"]
+= Pulling from private registries with delegated authentication
+
+A private registry can delegate authentication to a separate service. In these
+cases, image pull secrets must be defined for both the authentication and
+registry endpoints.
+
+////
+Is this relevant?
+[NOTE]
+====
+Third-party images in the Red Hat Container Catalog are served from
+`registry.redhat.io`. This registry delegates authentication to `sso.redhat.com`,
+so the following procedure applies.
+====
+////
+
+.Procedure
+
+. Create a secret for the delegated authentication server:
++
+----
+$ oc create secret docker-registry \
+    --docker-server=sso.redhat.com \
+    --docker-username=developer@example.com \
+    --docker-password=******** \
+    --docker-email=unused \
+    redhat-connect-sso
+
+secret/redhat-connect-sso
+----
++
+. Create a secret for the private registry:
++
+----
+$ oc create secret docker-registry \
+    --docker-server=privateregistry.example.com \
+    --docker-username=developer@example.com \
+    --docker-password=******** \
+    --docker-email=unused \
+    private-registry
+
+secret/private-registry
+----
+
+////
```

Comment:
I have this note commented out because I don't think "The Red Hat Connect Partner Registry `registry.connect.redhat.com`" exists any more. Is this correct? (same question as previous comment)

If so I'll need to remove the notes in this module *and* from 3.x.